### DISCUSSION TITLE
Portal 2 has received a update after a full decade

### DISCUSSION THREAD

**Discussion by genestrashcan** [BEEmod/BEE2-items#4180] [General]
Just a heads up Portal 2 has just received a major update after a full decade which changes some stuff in the puzzle maker

here is a link to the details

[update link](https://game-updates.info/portal2?52)

**Comment by vrad-exe** [comment]
Yes, we're well aware of this already. This isn't a discussion forum, please don't open an issue unless you actually have a bug report or feature request.

**Reply by Konclan** [reply]
Well it is now :P
